src/amazonsqs/amazonsqs_connector.bal

```diff
@@ -59,11 +59,11 @@ public type Client client object {
     # + queueName - Name of the queue to be created 
     # + attributes - Other attribute parameters 
     # + return - If success, URL of the created queue, else returns error
-    public remote function createQueue(string queueName, map<string> attributes) returns @tainted string|OperationError{
-        string amzTarget = "AmazonSQSv20121105.CreateQueue";
+    public remote function createQueue(string queueName, map<string> attributes) returns @tainted string|ErrorOperation{
+        string amzTarget = AMAZON_SQS_API_VERSION + "." + ACTION_CREATE_QUEUE;
         string endpoint = "/";
         string payload =  "";
```

Comment:
Can't we get the payload string creation part to one line? For clarity you can indent the code to multiple lines with each new line starting with the `+`

Reply:
Moved the payload creation part to a new method.